aopy/preproc/oculomatic.py

```diff
@@ -1,6 +1,7 @@
+from .. import precondition
 from .. import data as aodata
 
-def parse_oculomatic(data_dir, files, debug=True):
+def parse_oculomatic(data_dir, files, samplerate=480, debug=True):
```

Comment:
Why choose an oculomatic sampling rate of 480? It might be useful to default to the sampling rate of other data